src/RecoveryServices/RecoveryServices/Vault/SetAzureRmRecoveryServicesVaultContext.cs

```diff
@@ -24,7 +24,6 @@ namespace Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets
     /// <summary>
     /// Used to set RecoveryServices Vault Context
     /// </summary>
-    [CmdletDeprecation("This command is getting deprecated. Please use -VaultID parameter in all Recovery Services commands going forward.")]
```

Comment:
Will you confirm you want to withdraw the deprecation of  this cmdlet?

Reply:
Yes We want to withdraw the deprication, as we are no longer depricating this cmdlet.